src/scripts/website-blocking/WebsiteBlocking.ts

```diff
@@ -3,7 +3,11 @@ import browser from 'webextension-polyfill';
 import WebsiteBlockingApi from "./WebsiteBlockingApi";
 import WebsiteBlockingConverter from "./WebsiteBlockingConverter";
 
-const db = browser.extension.getBackgroundPage().db;
+const db = () => browser.extension.getBackgroundPage().db;
```

Comment:
what's the difference? it isn't fully initialised here?

Reply:
I think it's too early when I use it in backgorund.js, it was failing.